src/domain/resource/utils.js

```diff
@@ -87,3 +95,112 @@ export const isFree = (resource) => {
 
   return price === 0;
 };
+
+export const getOpeningHours = (resource, date = null) => {
+  if (!resource) {
+    return null;
+  }
+
+  const openingHours = get(resource, 'opening_hours', []);
+  if (!openingHours.length) {
+    return null;
+  }
+
+  const index = date ? findIndex(openingHours, item => item.date === date) : 0;
+  return {
+    closes: get(openingHours, `[${index}].closes`),
+    opens: get(openingHours, `[${index}].opens`),
+  };
+};
+
+const reservingIsRestricted = (resource, date) => {
+  if (!date) {
+    return false;
+  }
+
+  const isAdmin = get(resource, 'user_permissions.is_admin', false);
+  const isLimited = resource.reservable_before && moment(resource.reservable_before).isBefore(moment(date), 'day');
+
+  return isLimited && !isAdmin;
+};
+
+export const getOpenReservations = (resource) => {
+  return filter(resource.reservations, (reservation) => {
+    return reservation.state !== 'cancelled' && reservation.state !== 'denied';
+  });
+};
+
+export const getAvailabilityDataForNow = (resource, date) => {
+  const openingHours = getOpeningHours(resource, date);
+  const reservations = getOpenReservations(resource);
+
+  if (!openingHours || !openingHours.closes || !openingHours.opens) {
+    return { status: 'closed', bsStyle: 'danger' };
+  }
+
+  const nowMoment = moment();
+  const opensMoment = moment(openingHours.opens);
+  const closesMoment = moment(openingHours.closes);
+  const beginMoment = nowMoment > opensMoment ? nowMoment : opensMoment;
```

Comment:
AFAIK calling `moment()` returns an object, therefore, using `>` for comparison won't do the trick. 
I think it's better to use `isBefore()`/`isAfter()`

Reply:
This is basically copy/pasted from the old utility functions, it's just modified to work without redux/data straight from the API.